src/Prometheus.Client/Extensions/CounterExtensions.cs

```diff
@@ -1,7 +1,7 @@
 using System;
-using Prometheus.Client.Tools;
+using Prometheus.Client.Abstractions;
 
-namespace Prometheus.Client.Abstractions
+namespace Prometheus.Client.Extensions
```

Comment:
It's better to left such classes in Abstractions or move to root namespace. I would like to have them available without adding extra using,  otherwise end-user never find this extensions